src/control/cmd/daos_agent/procmon.go

```diff
@@ -194,15 +259,22 @@ func (p *procMon) handleRequests(ctx context.Context) {
 			return
 		case request := <-p.request:
 			switch request.action {
-			case drpc.MethodGetAttachInfo:
-				p.handleProcessAttach(ctx, request)
+			case drpc.MethodPoolConnect:
+				p.handlePoolConnect(ctx, request)
+			case drpc.MethodPoolDisconnect:
+				p.handlePoolDisconnect(request)
 			case drpc.MethodDisconnect:
 				p.handleProcessDisconnect(request)
 			default:
 				p.log.Errorf("Received request with invalid action type %s", request.action)
 			}
 		case resp := <-p.response:
 			p.log.Debugf("Received response from Process %d, terminated with %s", resp.pid, resp.err)
```

Comment:
Is this an actual error?  I can't quite tell from the verbiage.

Reply:
Maybe it isn't an error. I'll mark it Debugf instead since there is nothing we can really do about it.